native_client/javascript/Makefile

```diff
@@ -42,7 +42,9 @@ node-wrapper: copy-deps build
 	$(NODE_BUILD_TOOL) $(NODE_PLATFORM_TARGET) $(NODE_RUNTIME) $(NODE_ABI_TARGET) $(NODE_DIST_URL) package $(NODE_BUILD_VERBOSE)
 
 npm-pack: clean package.json index.js
-	npm install node-pre-gyp@0.14.x
+	npm install --prefix=$(NPM_ROOT)/../ node-pre-gyp@0.14.x typescript@3.6.2 @types/argparse@1.0.38
+	tsc --version
```

Comment:
Don't think this is necessary

Reply:
This is required since running `tsc` shows some error that I have no idea how to fix and silence with `|| true`, so I need this version check to ensure the `tsc` binary is properly installed.